readme.md

```diff
@@ -27,6 +30,8 @@ This plugin is still pretty much a work-in-progress.
 - Section movements!
 - And more...
 
+Check the [GitHub wiki][1] for instructions.
+
```

Comment:
It's a bit confusing being here, how about we move this to it's own section at the end? Like this for example:

```markdown
#### Usage
See the project's [wiki][1] for more information.
```